### PR TITLE
Prepend manifest.json configuration and update min requirement to symfony 3.4

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Prepend manifest.json configuration in admin and move into own asset package.

#### Why?

To has no conflicts with user specific configuration when they want to use an own manifest.json.

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php [hunk 1]
```diff
@@ -21,8 +22,29 @@
...
+    public function prepend(ContainerBuilder $container)
+    {
+        if ($container->hasExtension('framework')) {
+            $container->prependExtensionConfig(
+                'framework',
+                [
+                    'assets' => [
+                        'packages' => [
+                            'admin' => [
+                                'json_manifest_path' => '%kernel.project_dir%/web/adminV2/manifest.json',
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php]: path has changed `%kernel.project_dir%/web/admin/build/manifest.json`

File: src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php [hunk 2]
```diff
@@ -21,8 +22,29 @@
...
+     */
+    public function prepend(ContainerBuilder $container)
+    {
+        if ($container->hasExtension('framework')) {
+            $container->prependExtensionConfig(
+                'framework',
+                [
+                    'assets' => [
+                        'packages' => [
+                            'admin' => [
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php]: Rename to `sulu_admin`

**Comment by danrot** [comment]
@alexander-schranz Can you please also move it into the correct column in the project board?

File: composer.json [hunk 3]
```diff
@@ -55,7 +55,7 @@
         "symfony/filesystem": "^3.0",
         "symfony/finder": "^3.0",
         "symfony/form": "^3.0",
-        "symfony/framework-bundle": "^3.0",
+        "symfony/framework-bundle": "^3.3",
```
Comment (alexander-schranz) [inline composer.json]: The manifest based versioning is only available on https://symfony.com/blog/new-in-symfony-3-3-manifest-based-asset-versioning /cc @danrot @chirimoya

File: composer.json [hunk 4]
```diff
@@ -39,41 +39,41 @@
         "pagerfanta/pagerfanta": "^1.0.4",
         "piwik/device-detector": "^3.7",
         "ramsey/uuid": "^3.1",
-        "sensio/framework-extra-bundle": "^3.0",
+        "sensio/framework-extra-bundle": "^5.0",
```
Comment (alexander-schranz) [inline composer.json]: @danrot @chirimoya oriented by [symfony-standard](https://github.com/symfony/symfony-standard/blob/3.4/composer.json#L22) updated the framework bundle to 5.0

File: composer.json [hunk 5]
```diff
@@ -39,44 +39,44 @@
...
+        "symfony/templating": "^3.4",
+        "symfony/translation": "^3.4",
+        "symfony/twig-bridge": "^3.4",
+        "symfony/twig-bundle": "^3.4",
+        "symfony/validator": "^3.4",
+        "symfony/web-link": "^3.4",
         "twig/twig": "^1.11 || ^2.0",
         "willdurand/hateoas-bundle": "^1.1",
-        "sensio/distribution-bundle": "~5.0.6"
+        "sensio/distribution-bundle": "~5.0.19"
```
Comment (alexander-schranz) [inline composer.json]: updated by requirement of https://github.com/symfony/symfony-standard/blob/v3.4.6/composer.json#L21

File: composer.json [hunk 6]
```diff
@@ -39,44 +39,44 @@
...
-        "symfony/twig-bridge": "^3.0",
-        "symfony/twig-bundle": "^3.0",
-        "symfony/validator": "^3.0",
-        "symfony/web-link": "^3.0",
+        "symfony/property-access": "^3.4",
+        "symfony/proxy-manager-bridge": "^3.4",
+        "symfony/routing": "^3.4",
+        "symfony/security": "^3.4",
+        "symfony/security-bundle": "^3.4",
+        "symfony/swiftmailer-bundle": "^2.6.4",
```
Comment (alexander-schranz) [inline composer.json]: use also the same as the symfony standard 3.4 https://github.com/symfony/symfony-standard/blob/v3.4.6/composer.json#L25

File: composer.json [hunk 7]
```diff
@@ -125,7 +125,7 @@
     },
     "suggest": {
         "pulse00/ffmpeg-bundle": "Needed (as in require-dev) to generate preview images for videos",
-        "symfony/monolog-bundle": "^3.0",
+        "symfony/monolog-bundle": "^3.1",
```
Comment (alexander-schranz) [inline composer.json]: based on symfony-standard 3.4 requirement: https://github.com/symfony/symfony-standard/blob/v3.4.6/composer.json#L23

**Pull request merged**